### ISSUE TITLE
ambiguous jars (with different qualified names) found for open test alliance

### ISSUE THREAD

**Issue by akozlova** [junit-team/junit5#181]
I tried first with https://oss.sonatype.org/content/repositories/snapshots/org/junit/open-test-alliance/ but failed because really used is https://oss.sonatype.org/content/repositories/snapshots/org/opentest4j/opentest4j/1.0.0-SNAPSHOT/ 

If that's correct, could you please mark the first one as outdated or hide it.

**Comment by sbrannen** [comment]
@akozlova, the artifact in the `org.junit` group is the one we relied on during the prototype, but it is no longer valid.

The reason it resides under `org.junit` is that we did not have a stand-alone project for the OTA at that point in time. Since then, however, we have created a stand-alone project for the OTA (outside of JUnit). Thus the OTA jar now has its own group ID.

I hope that clarifies the status quo.

> If that's correct, could you please mark the first one as outdated or hide it.

I'm not a Maven Central guru, but I don't believe it's possible or even a good idea to remove any artifact from Maven Central.

The reason is that any given artifact was valid at the time it was published. Any JUnit Lambda prototype code requires that artifact. We of course don't expect people to work with JUnit Lambda going forward; however, there are in fact blogs and tutorials that are based on the prototype.

So, in my opinion, the prototype artifact needs to remain in Maven Central as is. With that in mind, I am closing this issue. However, feel free to continue the discussion here if you are unsatisfied with my explanation.

Cheers,

Sam
